{{cookiecutter.project_dirname}}/Makefile

```diff
@@ -99,6 +103,30 @@ remote: pip_update  ## Install remote requirements and dependencies
 report:  ## Run coverage report
 	./scripts/report.sh
 
+.PHONY: runserver
+runserver:  ## Django run
+	python3 -m manage runserver 0:8000
+
+.PHONY: scan_docker_image
+scan_docker_image:  ## Scan docker image vulnerabilities
+	docker run --rm \
+	--user=$$UID:`id -g` \
+	--volume /var/run/docker.sock:/var/run/docker.sock \
+	--volume $$PWD/.trivy:/app/.trivy \
+	--name trivy-{{ cookiecutter.project_slug }}-backend \
+	aquasec/trivy image \
+	--no-progress \
+	--cache-dir .trivycache/ \
+	--format template --template "@contrib/html.tpl" \
+	--output "/app/.trivy/report.html" \
+	--security-checks vuln \
+	--severity CRITICAL,HIGH,MEDIUM,LOW \
+	{{ cookiecutter.project_slug }}_backend:latest
+
+.PHONY: scan_terraform
+scan_terraform:  ## Scan terraform vulnerabilities
```

Comment:
remove this one, will be added in separate PR

Reply:
okk